tests/helpers/worker.py

```diff
@@ -80,7 +85,131 @@ class KSExecuteActivityAction:
     wait_for_cancellation: Optional[bool] = None
     heartbeat_timeout_ms: Optional[int] = None
     retry_max_attempts: Optional[int] = None
-    non_retryable_error_types: Optional[Iterable[str]] = None
+    non_retryable_error_types: Optional[Sequence[str]] = None
+
+
+@workflow.defn(name="kitchen_sink")
```

Comment:
Are you keeping this kitchen sink workflow around long term?

Reply:
Not sure yet. It powers client and activity tests since those were written prior to workflows even being available. It's pretty small for what it does.